model/system/registry.yaml

```diff
@@ -22,6 +22,12 @@ groups:
             - id: used
               value: 'used'
               stability: development
+              brief: >
```

Comment:
nit: could you please brief concise and move additional info to the note?

Reply:
Fixed: https://github.com/open-telemetry/semantic-conventions/pull/2534/commits/ae308aef63c6f84f42a346e2280c5874594cf5c4